product-service/get-products-by-id.js

```diff
@@ -1,28 +1,66 @@
-import productList from "./productList.json";
+import {Client} from 'pg';
+
+const {PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD} = process.env,
+  dbOptions = {
+    host: PG_HOST || 'localhost', 
+    port: PG_PORT || 5432,
+    database: PG_DATABASE,
+    user: PG_USERNAME || 'postgres',
+    password: PG_PASSWORD,
+    ssl:{
+      rejectUnauthorized: false
+    },
+    connectionTimeoutMillis: 5000
+  };
```

Comment:
I see you don't care about reuse code. I know that is not a part of Task requirements but it is an obvious moment

Reply:
I see you don't care about politeness ;)

Reply:
Sorry, didn't want to offend you. It was just a recommendation....